pytest_tests/tests/object/test_object_api.py

```diff
@@ -592,6 +592,52 @@ def test_object_get_range(self, request: FixtureRequest, storage_objects: list[S
                         == range_content
                     ), f"Expected range content to match {range_cut} slice of file payload"
 
+    @allure.title("Validate native object API get_range for a complex object")
+    @pytest.mark.grpc_api
+    def test_object_get_range_complex(self, default_wallet: NodeWallet, container: str, complex_object_size: int):
+        """
+        Validate get_range for object by native gRPC API for a complex object
+        """
+        file_path = generate_file(complex_object_size)
+        oid = put_object_to_random_node(
+            default_wallet.path,
+            file_path,
+            container,
+            shell=self.shell,
+            neofs_env=self.neofs_env,
+        )
+
+        file_ranges_to_test = []
+
+        parts = get_object_chunks(default_wallet.path, container, oid, self.shell, self.neofs_env)
+
+        # range is inside one child
+        file_ranges_to_test.append((0, parts[0][1] - 1))
+        # range matches child
+        file_ranges_to_test.append((parts[0][1], parts[1][1]))
+        # range requires more than one child (it includes OR does not include the first and the last children)
```

Comment:
in the issue, i meant both cases (a one when range includes end pieces and a one when does not). currently, this comment sounds like it does not know what is happening

Reply:
I updated comments and cases a little bit